test/__init__.py

```diff
@@ -150,22 +150,27 @@ def add_file(self, file):  # type: (File) -> UserModule
     def url(self):  # type: () -> str
         return os.path.join('s3://', self.bucket, self.key)
 
-    def upload(self):  # type: () -> UserModule
-        with _files.tmpdir() as tmpdir:
-            tar_name = os.path.join(tmpdir, 'sourcedir.tar.gz')
-            with tarfile.open(tar_name, mode='w:gz') as tar:
-                for _file in self._files:
-                    name = os.path.join(tmpdir, _file.name)
-                    with open(name, 'w+') as f:
+    def create_tar(self, dir=os.path.dirname(os.path.realpath(__file__))):
```

Comment:
Avoid using `dir`, it is a Python built-in function name.
Let's create a variable or constant instead of passing the Python function directly here, example: https://github.com/aws/sagemaker-containers/blob/d782e869e130e9067d5152047da872a50f399714/test/functional/test_training_framework.py#L30

Reply:
will do